react.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
             extensions: ['.js'],
         }],
         'react/jsx-first-prop-new-line': ['error', 'multiline'],
-        'react/jsx-handler-names': 'error',
+        'react/jsx-handler-names': 'off',
```

Comment:
updated. changed to this setting instead. this will throw an error if the eventHandler is not prefixed by 'on'.
```
'react/jsx-handler-names': ['error', {
       eventHandlerPrefix: 'on',
}]
``